### DISCUSSION TITLE
new tool: www.fastflood.org

### DISCUSSION THREAD

**Discussion by bastianvandenbout** [bastianvandenbout/LISEM#14] [Announcements]
Dear users, 

we have been developing a new super-fast flood simulation method. It gets to very high accuracy, but 1500x faster by only predicting peak flow heights (flood hazard maps), and not the full dynamic progress of the event.
You can find the preprint here: https://eartharxiv.org/repository/view/4777/ 

We have been working on making this available in the browser
see: [www.fastflood.org.](http://www.fastflood.org)

You can automatically download elevation/land ue/infiltration and rianfall data, and quickly estimate flood maps.
It is fast enough for interactive editing and mitigation design.

Not all the features are fully working yet, its still recieving updates, but if you are interested in testing this, its available.

A video-tutorial will be coming soon!
Hope you find it usefull,

Bastian

**Comment by feefladder** [comment]
I was thinking that it is not entirely necessary to host all maps for the pro version, since you could also get them from a COG right?

**Reply by bastianvandenbout** [reply]
Hi, thank you for the suggestion!
This might be possible, although for global datasets the file sizes would be in the terabytes.
Besides that, I havent found such such files for datasets available globally

If you know of any COG that are available with relevant data I might be able to give it a try
But in principle you can always also load your own geotiff files for various data inputs in fastflood.org

**Reply by feefladder** [reply]
The point of COGs is not to keep an entire copy of the dataset, but rather to download only the area of interest on-demand using e.g. `GDAL translate`
For speed, you could use a caching mechanism (e.g. redis) to run the analysis, then limit the file size per IP address or something so that people don't run LISEM for the entire earth on your server, but I think you already do that :).

For example [copernicus dem](https://registry.opendata.aws/copernicus-dem/) but also IsdaSoil land cover etc is available, which has a dem as covariate.
[here](https://medium.com/nerd-for-tech/soil-and-agronomy-data-cube-for-africa-at-30-m-spatial-resolution-591e04bfc372) is a blog post and [here](https://isdasoil.s3.amazonaws.com/index.html?t=catalogs) is the STAC browser for it.
